src/extension/exponent/exponentHelper.ts

```diff
@@ -41,8 +45,8 @@ export class ExponentHelper {
 
     public configureExponentEnvironment(): Q.Promise<void> {
         this.lazilyInitialize();
-        this.logger.info("Making sure your project uses the correct dependencies for exponent. This may take a while...");
-        this.logger.logStream("Checking if this is Expo app.");
+        this.logger.info(localize("MakingSureYourProjectUsesCorrectExponentDependencies", "Making sure your project uses the correct dependencies for exponent. This may take a while..."));
```

Comment:
```suggestion
        this.logger.info(localize("MakingSureYourProjectUsesCorrectExponentDependencies", "Making sure your project uses the correct dependencies for Expo. This may take a while..."));
```